content/quickstart.md

```diff
@@ -39,8 +83,74 @@ Under construction.
 
 ## Loading your model
 
-Under construction.
+#### First time
+
+1- open the ``model`` menu
+
+2- load model
+
+3- open ``.vrm`` file
+
+#### Loading a used model
+
+1- open ``preset`` menu
+
+2- select preset and hit ``load``
+
+## Tracking
+
+### OpenSeeFace
+
+Runs using your webcam on laptop/pc. Source code provided by [emilianavt](https://github.com/emilianavt/OpenSeeFace)
+
+### IFacialMocap
+
+Uses the 3D tracking capabilities of Apple Inc devices to provide optimal tracking data from your phone. Use if your model has
+perfect sync.
+
+iFacialMocap is an app that captures facial movements with an iPhone equipped with FaceID. This app sends the captured motion in real time to 3DCG software such as Maya, Unity, Blender on PC. -IFM developers

```

Comment:
Quote should be encased in double quotes. Would be better to leave it out since this could go out-of-date very quickly

"IFM" should be expanded to iFacialMocap since the acronym has not been defined anywhere else on the page